javalib/src/main/scala/java/util/concurrent/ConcurrentSkipListSet.scala

```diff
@@ -8,10 +8,7 @@ import scala.math.Ordering
 import scala.collection.JavaConversions._
 
 class ConcurrentSkipListSet[E] protected (ordering: Ordering[_ >: E], _comparator: Comparator[_ >: E])
-    extends AbstractSet[E]
-    with NavigableSet[E]
-    with Cloneable
-    with Serializable { self =>
+    extends TreeSet[E](ordering, _comparator) {
```

Comment:
This will not work properly. You Can't change the class dependencies of classes that already exist in java. One problem that might occure is:
```scala
val set = new ConcurrentSkipListSet(...)
if (set.isInstanceOf[TreeSet]) {
  // will execute this in on JS
} else {
  // will execute this on JVM
}